### ISSUE TITLE
Clr-installer

### ISSUE THREAD

**Issue by cgamez** [clearlinux/distribution#837]
Hi, I have tried to install Clear Linux on barebones with dual boot option (triple boot?), Clear, Fedora and Windows.  As first OS, Clear Linux is the way, if I understand well.  If I try the clr-installer (text), it will not show all the installation steps as the web manual suggests, specially the advanced options.  I manually tried by creating partitions and even using the gui-installer, but the gui installer always installs on the free disk space instead of using the created partitions.  I have not tried to install any other OS yet, but before trying myself other options, I would like to know if someone can recommend me how to do the dual/triple install with the current installers.  I do not mind a different approach. Thanks in advance.

**Comment by cgamez** [comment]
I have tried a different road... I installed Win 10 first and left 64GB free space for Clear-Linux.  Unfortunately after 1.5 hours of installation, the installer exited with code status 35.  See attached pictures of exit message and partitions 5, 6 and 7 were created automatically by Clear-Linux Installer.
![IMG_20190602_175554997](https://user-images.githubusercontent.com/31857373/58768507-f4fc2d00-8561-11e9-8d38-803791c7b855.jpg)
![IMG_20190602_180024085](https://user-images.githubusercontent.com/31857373/58768508-f9284a80-8561-11e9-81d2-52bf30fa27b7.jpg)

**Comment by mdhorn** [comment]
@cgamez First, I'd suggest getting a more recent version of the OS and the installer. 2.0.3 is several weeks old and the latest release finally has much-improved status update; we were waiting for an update on an underlying tool which is now released. Though there are other fixes to, it will mainly fix the progress update aspect.

Exit status 35 from swupd means: Unable to fix/replace/delete one or more files.
I'd need more details logs to know what happened there.
You can enable detailed debugging logs with either adding `clri.loglevel=4` to the boot command line or by starting the installer from a terminal with: `sudo -E clr-installer-gui -l 4`

We do have documentation on dealing with Dual Booting:
https://clearlinux.org/documentation/clear-linux/tutorials/multi-boot#multi-boot
But looks like it some updates are needed for the new installer.

In general, the installer will only use unallocated disk space (free space) to ensure it doesn't overwrite something. I think you were on the right path leaving free space for the installer to use.
Though, to be more in line with what I see in our documentation, you should try installing Clear Linux* OS first.

- Leave only the free space for Clear Linux on the disk
  - Allocate (reserve) the space for Windows by creating an NTFS 
- Install Clear Linux OS; it will use the free space
- Remove the pre-allocated NTFS so it is free for Windows to use for it's install.

**Comment by cgamez** [comment]
@mdhorn Thanks, I will try your recommendations.  I downloaded the iso a few weeks ago.  How can I check or have the latest installer from?

**Comment by mdhorn** [comment]
@cgamez The latest version is published on the Clear Linux downloads site:
https://clearlinux.org/downloads

**Comment by cgamez** [comment]
@mdhorn Now I understand.  Yesterday I downloaded the installer twice: in the morning and in the night and both were different versions, so it is updated quite often.  With clear-29790 I was able to install by shrinking the Windows Partition and booting it via Windows UEFI boot manager.  I'm a bit dissapointed that my celeron 6W CPU netbook (4GB RAM) is extremely fast and my core i5HQ 45W laptop (16GB RAM) is really slow.  I did not used it much, it was already late, both are desktop versions.  Thanks for your help.
